cmd/kubeadm/app/preflight/checks_test.go

```diff
@@ -849,3 +849,24 @@ func TestNumCPUCheck(t *testing.T) {
 		})
 	}
 }
+
+func TestMemCheck(t *testing.T) {
```

Comment:
this test will ONLY pass on Linux. should we remove? @neolit123 @rosti

Reply:
You can skip it if the `GOOS != "linux"`

Reply:
done